MapboxVision/Services/ModelPerformance.swift

```diff
@@ -4,7 +4,13 @@ import UIKit
 /**
  Enumeration which determines whether SDK should adapt its performance to environmental changes (acceleration/deceleration, standing time) or stay fixed.
  */
-public enum ModelPerformanceMode {
+public enum ModelPerformanceMode: Comparable {
+    /// :nodoc:
+    public static func < (lhs: ModelPerformanceMode, rhs: ModelPerformanceMode) -> Bool {
+        if lhs == rhs { return false }
+        return lhs == .dynamic
```

Comment:
As we discussed it makes sense to add a short comment describing the semantics of comparison.

Reply:
Added, PTAL.